.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  tags:
+    include:
+      - "*"
+pr:
+  branches:
+    include:
+      - "*"
+
+jobs:
+  - job: Build
+    displayName: "Build"
+
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - task: NodeTool@0
+        displayName: "Install node"
+        inputs:
+          versionSpec: "12.x"
+
+      - task: Cache@2
+        displayName: "Check cache"
+        inputs:
+          key: "yarn | yarn.lock"
+          path: "node_modules"
+          cacheHitVar: CACHE_RESTORED
+
+      - script: "apt-get install -y rpmbuild"
+        displayName: "Install apt dependencies"
+
+      - script: "yarn install --frozen-lockfile --non-interactive"
```

Comment:
non-interactive shouldn't be required here

Reply:
Better safe than sorry

Reply:
I haven't tried yet how it behaves if you specify an non-existing version in CI